src/sentry/wsgi.py

```diff
@@ -20,12 +20,15 @@
 # Run WSGI handler for the application
 application = WSGIHandler()
 
+
+warmup_host_name = settings.ALLOWED_HOSTS[0] if settings.ALLOWED_HOSTS[0] != "*" else "127.0.0.1"
```

Comment:
there should be a test showing this behaviour or it's just going to regress

Reply:
added a test in https://github.com/getsentry/sentry/pull/89361/commits/940f34f44afbeea9932279d38f0ff90a47407505